### ISSUE TITLE
Refactor

### ISSUE THREAD

**Issue by otacke** [otacke/h5p-advent-calendar#11]
As a private weekend hackathon, this got out of hand a little :-) Things to consider:

- Use SASS for stylesheets (plenty of colors codes, etc.)
- general code cleaning

**Comment by JohnSmith-LT** [comment]
Hi @otacke I intended to fork this and do some development on it to make it a bit more generic (date wise) and also to add in more types of content... But then I noticed on Lumi that there is a Content Calendar that appears to be very much based on this Calendar type yet I cannot find it available anywhere to clone?

Did you develop the Content Calendar also and/or do you know if the source is available to work on or is it closed source?

**Comment by otacke** [comment]
I assume you mean https://github.com/otacke/h5p-content-calendar It's not based on AdventCalendar at all though.

**Comment by JohnSmith-LT** [comment]
Ah, I get a 404 when I go to that URL and I looked through your repos and it didn't come up. Is it private?

**Comment by otacke** [comment]
Yes, it's still private (sorry, had not checked - too many projects), because I have not yet published any information. Let me check Lumi. I'd be curious where they got hold of it. Potentially my client provided the archive.

**Comment by otacke** [comment]
Yeah, that's the one on Lumi Cloud. They also already have the "Phrase Randomizer" :-) Don't know if they have the latest versions though. I may have tweaked both after delivery.

**Comment by JohnSmith-LT** [comment]
Ah, ok, no worries. I might just fork this Advent one and work on it just now then if the two are not similar, I like the Advent layout, just need it for different dates...

**Comment by otacke** [comment]
Feel free to change whatever you want in the code. That is what open source is for. I won't change anything related to dates in Advent Calender, because it is what it is called :-)

**Comment by JohnSmith-LT** [comment]
Sorry, last thing I meant to ask, that I was confused about initially. I tested the Content Calendar one on Lumi and then exported it to import into my development environment but the images didn't comedown with it, just the javascript and json. I then read that on Lumi they have offloaded images and audio (and presumably video) to AWS.

So is that then intended behaviour - when you choose reuse and download he .h5p file that offloaded content is not downloaded with the package or is that a bug?

**Comment by otacke** [comment]
This is not intended behavior from an H5P perspective, no. Exported content should contain everything that is required to run the content on any H5P integration (there are some exceptions that don't apply here). If it does not, that's a deficiency of Lumi Cloud. You'll have to ask Lumi about details.

**Comment by JohnSmith-LT** [comment]
Thanks. I didn't think so and wondered if it was due to their modifications. I'll take it up with them because the offloading may help them and their contend delivery but then having the download option isn't really helping the whole H5P community as a whole by working that way... Cheers @otacke, have a great weekend!

**Comment by otacke** [comment]
You're welcome. Feel free to reach out via email (see https://github.com/otacke) for more questions. Might make more sense to use issue comments on github.
